src/Components/Categories.js

```diff
@@ -105,10 +104,12 @@ export default function Categories() {
           <Card.Header className="card-component card-heading">
             Categories
           </Card.Header>
+
           <Button onClick={handleShowCategoriesNew}>
             <Button>Add Category</Button>
           </Button>
```

Comment:
I think this is unrelated to this PR/issue, but I noticed this error in the console:
```
index.js:1 Warning: validateDOMNesting(...): <button> cannot appear as a descendant of <button>.
    in button (created by Button)
    in Button (at Categories.js:109)
    in button (created by Button)
    in Button (at Categories.js:108)
    in div (created by Card)
    in Card (at Categories.js:103)
    in div (at Categories.js:102)
    in div (at Categories.js:101)
    in Categories (at PrivateRoute.js:14)
    in Route (at PrivateRoute.js:10)
    in PrivateRoute (at App.js:127)
    in Switch (at App.js:70)
    in Router (created by BrowserRouter)
    in BrowserRouter (at App.js:67)
    in div (at App.js:66)
    in App (at src/index.js:19)
    in CurrentOrganizationProvider (at src/index.js:17)
    in CurrentUserProvider (at src/index.js:16)
    in StrictMode (at src/index.js:15)
```